### PR TITLE
dev/core#5439 FIx access help for Smarty3+

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
dev/core#5439 FIx access help for Smarty3+
https://lab.civicrm.org/dev/core/-/issues/5439

Before
----------------------------------------

After
----------------------------------------
image

Technical Details
----------------------------------------

Comments
----------------------------------------
@braders

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Core/Page/Inline/Help.php [hunk 1]
```diff
@@ -52,4 +53,12 @@ public function run() {
     }
   }
 
+  private function getAccessKey(): string {
+    $ua = strtolower($_SERVER['HTTP_USER_AGENT'] ?? '');
+    if (str_contains($ua, 'mac')) {
+      return '<span>CTRL</span>+<span>ALT</span>';
```
Comment (braders) [inline CRM/Core/Page/Inline/Help.php:59]: Looks good. I'm just wondering now if we should take the opportunity to change `ALT` to `Option` on the mac condition, as that's the language mac keyboards almost always use (and would match what's documented as the keyboard combination here: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey)

Comment (eileenmcnaughton) [reply CRM/Core/Page/Inline/Help.php:59]: So would we simply change the word `ALT` above to `OPTION` (I no longer have a mac)

Comment (braders) [reply CRM/Core/Page/Inline/Help.php:59]: Yes, exactly.

Comment (colemanw) [inline CRM/Core/Page/Inline/Help.php:59]:
```suggestion
      return '<span>CTRL</span>+<span>OPTION</span>';
```

**Pull request merged**